### DISCUSSION TITLE
Login Problem to Vercel

### DISCUSSION THREAD

**Discussion by morteza-araby** [vercel/community#3132] [Help] [answered]
### Summary

Hi,
Every time I try to login with my email address I get prompt to enter phone number and when I do that I get,

'This phone number is already associated with a Vercel account.'

This email and phone number are associated to my login credentials. So what is the porblem?

Thanks
Morteza

### Steps to Reproduce

1- Browse to vercel.com
2- login
3- Continue with Email
4- input my email address
5- Click on Continue with Email
6- In verification dialog, I enter my phone number

Then error 'This phone number is already associated with a Vercel account' appears, and I cannot login

**Comment by davidxiang101** [comment]
I'm encountering the same problem.

**Comment by amyegan** [comment]
Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team. You can privately share the email address and phone number there so our team can look into this problem for you 🙇

**Reply by davidxiang101** [reply]
Hi, I tried submitting a case, but it requires me to login. So I ran into the same problem as above. Is there another way to privately share my account details?

**Reply by amyegan** [reply]
You can open a case for this login issue by emailing [registration@vercel.com](mailto:registration@vercel.com) with the relevant info.

**Reply by hastinulchotimah** [reply]
I've sent the email, why hasn't there been any further notification?

**Reply by programmablemike** [reply]
I'm having the same issue as the user above. Sent multiple emails spaced over several days to the registration email account without any response.

**Reply by amyegan** [reply]
If you can let me know your case number, I can check on the ticket for you.

**Reply by abdulrabbani00** [reply]
This is my case number @amyegan - 00180444

Similar issue as above.

**Reply by Hutuan6886** [reply]
please, check my ticket number: 00185420 
thank you so much

**Reply by aadium** [reply]
Please check my number: +97455632157

**Reply by SonBeoDeveloper** [reply]
plwase check my case number is 00208491 @amyegan

**Reply by ojyochan** [reply]
please, check my ticket number: 00210286 @amyegan

**Reply by ryunosuke-hayashi-private** [reply]
please, check my ticket number: 00212779 @amyegan

**Comment by Abbiasone** [comment]
hey sorry, when i'm trying to log in in vercel and continue with github it's says "This user account is blocked' there's some reasons?

**Comment by jasonthorsness** [comment]
I just got the same error; the weird thing is something in their system got far enough that I got the welcome email, but I can't actually log in to the site as every time I am blocked by that phone number prompt (even when I click "log in"). I followed up with [registration@vercel.com](mailto:registration@vercel.com) per the above instructions.

**Reply by jasonthorsness** [reply]
The team at registration@vercel.com took care of it already, thanks!

**Comment by maorazoulay** [comment]
**Sep 2023 Update:** The issue still persists. 

This is kinda embarrassing Vercel, you are supposed to be the leader in your field but then your website has this silly issue. 👎

**Comment by hastinulchotimah** [comment]
why every time I log in via email an error like this appears (We are unable to complete your signup. Please read this guide for further details.)

**Comment by harveybland** [comment]
I am getting the same error. I've sent an email to registration@vercel.com but I have had no reply for 3 days. How long do they take to respond?

**Comment by 999bits** [comment]
I sent an email to registration@vercel.com too. but not responded yet.

**Comment by harveybland** [comment]
What day did you send an email? I was just about to write another one about
it.

Glad it's not just me whos having the issue

**Comment by runtoearth** [comment]
I'm having the same issue and I've never even signed up for vercel until today.

**Comment by gapormann** [comment]
I can't access my Vercel account, a loop happens sending verification code and authentication is not carried out.

**Reply by 200anxy** [reply]
I am having the same problem, please help someone

**Comment by portlogics-leejiwon** [comment]
I'm having the same problem. Please check my number 00208501

**Comment by gabrielyotoo** [comment]
This is so nonsense. Why can't I have multiple accounts? should I have multiple phone numbers instead? lol

**Comment by amyegan** [comment, marked as answer]
Vercel employs multiple layers of security and verification during the account signup process to protect against platform abuse. If you received an error while signing up, please contact our registration team at [registration@vercel.com](mailto:registration@vercel.com)

**Reply by amyegan** [reply]
For more information, please check these pages:
- [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
- [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support)
- [Fair Use Guidelines](https://vercel.com/docs/limits/fair-use-guidelines#fair-use-guidelines) and [Terms of Service](https://vercel.com/legal/terms)
